bss/db/history_db.go

```diff
@@ -0,0 +1,136 @@
+package db
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"math/big"
+	"os"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type L2BlockEntry struct {
+	BlockNumber *big.Int    `json:"block_number"`
```

Comment:
Using `uint64` or `hexutil.Uint64` would also be fine, I don't think the block number will ever go over 52 or 64 bits.